src/modules/ekf2/EKF2.cpp

```diff
@@ -714,6 +714,9 @@ void EKF2::PublishAidSourceStatus(const hrt_abstime &timestamp)
 	// airspeed
 	PublishAidSourceStatus(_ekf.aid_src_airspeed(), _status_airspeed_pub_last, _estimator_aid_src_airspeed_pub);
 
+	// sideslip
+	PublishAidSourceStatus(_ekf.aid_src_sideslip(), _status_sideslip_pub_last, _estimator_aid_src_sideslip_pub);
```

Comment:
It might be much simpler to start publishing this stuff directly from the EKF backend. Then we could skip some of the boilerplate, not have to track last update timestamps, etc. The downside is we'd have to put it behind an ifdef to maintain the standalone unit test build.

Idea for later...